BasiliskII/src/Windows/router/tcp.cpp

```diff
@@ -1505,9 +1504,9 @@ static void init_tcp_listen_ports()
 	const char *port_str;
 	while ((port_str = PrefsFindString("tcp_port", index++)) != NULL) {
 		uint32 iface = 0;
-		char *if_str = strchr(port_str,',');
+		const char *if_str = strchr(port_str,',');
 		if(if_str) {
-			*if_str++ = 0;

```

Comment:
I think it was meant to null-terminate port_str at the comma position, for the call strtoul() below.